python/source/state_representation/bind_state.cpp

```diff
@@ -38,9 +38,8 @@ void state(py::module_& m) {
   c.def("is_empty", &State::is_empty, "Getter of the empty attribute");
   c.def("set_empty", &State::set_empty, "Setter of the empty attribute", "empty"_a=true);
   c.def("set_filled", &State::set_filled, "Setter of the empty attribute to false and also reset the timestamp");
-  c.def("get_timestamp", &State::get_timestamp, "Getter of the timestamp attribute");
-  c.def("set_timestamp", &State::set_timestamp, "Setter of the timestamp attribute");
-  c.def("reset_timestamp", &State::reset_timestamp, "Reset the timestamp attribute to now");
+  c.def("get_epoch", &State::get_epoch, "Getter of the time since last modification attribute");
```

Comment:
don't forget the python!

Reply:
I need to push before I can rebuild the tests for python...